src/skyparams.h

```diff
@@ -0,0 +1,64 @@
+/*
+Minetest
+Copyright (C) 2019 Jordach, Jordan Snelling <jordach.snelling@gmail.com>
+This program is free software; you can redistribute it and/or modify
+it under the terms of the GNU Lesser General Public License as published by
+the Free Software Foundation; either version 2.1 of the License, or
+(at your option) any later version.
+This program is distributed in the hope that it will be useful,
+but WITHOUT ANY WARRANTY; without even the implied warranty of
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+GNU Lesser General Public License for more details.
+You should have received a copy of the GNU Lesser General Public License along
+with this program; if not, write to the Free Software Foundation, Inc.,
+51 Franklin Street, Fifth Floor, Boston, MA 02110-1301 USA.
+*/
+
+#pragma once
+
+struct SkyboxParams
+{
+    video::SColor bgcolor;
+    std::string type;
+    std::vector<std::string> params;
+    bool clouds;
```

Comment:
wrong indent (also in the other structs)

Reply:
`cloudparams.h` does the exact same thing.

Reply:
Then it's also wrong.

Reply:
Indentation needs fixing.

Reply:
Resolved.